dotnet/test/Svg.test.cs

```diff
@@ -0,0 +1,19 @@
+namespace StackExchange.StacksIcons.Test;
+
+public class SvgTest
+{
+    [Fact]
+    public void FindsEmbeddedResources()
+    {
+        var icon = Svg.Accessibility;
+        var spot = Svg.Spot.Ads;
+        Assert.False(string.IsNullOrWhiteSpace(icon.Value));
+        Assert.False(string.IsNullOrWhiteSpace(spot.Value));
+    }
+
+    [Fact]
+    public void CombinedCacheBreaker()
```

Comment:
If we expose this to consumers perhaps we should say a couple of words about it in the readme?

Reply:
See above comment about this now being removed (no longer applicable)